server/src/main/java/org/apache/cloudstack/vm/UnmanagedVMsManagerImpl.java

```diff
@@ -1354,6 +1393,6 @@ public String getConfigComponentName() {
 
     @Override
     public ConfigKey<?>[] getConfigKeys() {
-        return new ConfigKey<?>[] { UnmanageVMPreserveNic };
+        return new ConfigKey<?>[]{UnmanageVMPreserveNic};
```

Comment:
is this needed?

Reply:
I think this might be from IntelliJ auto formatting from some rules I had setup. If this is against formatting conventions, I can revert it?